chcemvediet/apps/inforequests/cron.py

```diff
@@ -219,15 +218,22 @@ def publish_inforequests():
         try:
             for branch in inforequest.branches:
                 action = branch.last_action
-                days_to_publish = DAYS_TO_CLOSE_INFOREQUEST + DAYS_TO_PUBLISH_INFOREQUEST
-                days_behind_closed = (local_today() - local_date(action.created)).days
-                if (action.deadline
-                        and action.deadline.snooze_calendar_days_behind < days_to_publish):
-                    break
-                elif days_behind_closed < DAYS_TO_PUBLISH_INFOREQUEST:
-                    break
```

Comment:
@viliambalaz POZOR! to `elif` nebude fungovat tak ako chces, ved v prvom `if` mas `and` s dvomi podmienkami. A my chceme porovnavat `days_behind_closed` iba ked je `action.deadline` false. Lenze `elif` ti sposobi, ze do druhej vetvy sa dostanes aj ked `action.deadline` je true ale false bola podmienka za `and`. Takze ti to potom zverejni aj infoziadosti, ktore sice maju deadline s `snooze_calendar_days_behind < days_to_publish`, ale od poslednej akcie uz ubehlo `DAYS_TO_PUBLISH_INFOREQUEST`.

Reply:
@martinmacko47 Ano sorry, myslel som na to uz v predchadzjaucom commite ked som podmienku `if action.deadline is None` nechal samostatne, len som sa pomylil pri opravovani ked som to robil analogicky ako to bolo vo funkcii `close_inforequest`.